### PR TITLE
feature/AT-UN-UI 14

### PR DESCRIPTION
**Pull request by MajoYanez**
@fasaga

### CODE REVIEW

**Review by fasaga — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+NODE_ENV=development
```
Comment (at-internship) [inline .env.example:1]: .env file should be part of .gitignore, please remove this file

File: src/services/course.service.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 const courseService = {};
 
 courseService.getAllCourses = () => {
-    return axios({
-      method: "GET",
-      url: config.AT_UNIVERSITY_API_HOST + `/course`
-    });  
+  return axios({
+    method: "GET",
+    url: AT_UNIVERSITY_SERVICE_URI + `/course`
```
Comment (at-internship) [inline src/services/course.service.js]: should be:  `url: AT_UNIVERSITY_SERVICE_URI + `/v1/course``

Comment (at-internship) [reply src/services/course.service.js]:
```
const axios = require("axios");

const AT_UNIVERSITY_SERVICE_URI = process.env.AT_UNIVERSITY_SERVICE_URI;
console.log("AT_UNIVERSITY_SERVICE_URI:" + AT_UNIVERSITY_SERVICE_URI);

module.exports = {
    getAllCourses: () =>
        axios({
            method: "GET",
            url: AT_UNIVERSITY_SERVICE_URI + `/v1/course`,
            headers: {
                "content-type": "application/json",
            },
        }).catch(function(error) {
            console.log("Error: " + error.message);
        }),
};
```

**Review by at-internship — CHANGES_REQUESTED** [review]
Please take a look

File: .env.example (hunk 1 above)
Comment (fasaga) [reply .env.example:1]: It is a .example file, and its intention is to help the interns to know all the variables available in the system, they will copy this file to create their own .env file

**Review by at-internship — APPROVED** [review]
Approved

**Pull request merged**